src/components/common/WalletListHiddenItem.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from 'react'
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { SwipeRow } from 'react-native-swipe-list-view'
+
+import { WALLET_LIST_OPTIONS_ICON } from '../../constants/indexConstants.js'
+import { THEME } from '../../theme/variables/airbitz.js'
```

Comment:
Why are you using the old Theme model

Reply:
Because this is not yet the themed refactor of the wallet list modal